### DISCUSSION TITLE
Error on install - soft lockup - CPU#0 while applying salt

### DISCUSSION THREAD

**Discussion by CyberBitz** [Security-Onion-Solutions/securityonion#1634] [Unsupported Versions]
![image](https://user-images.githubusercontent.com/582721/97042539-95803200-1547-11eb-86d2-7f82e1554ef5.png)

I tried on 2.4 and this time on 2.4.1 same issue during salt install. 100 GB HDD and 2/2 CPU/Cores 16 GB Ram.

**Comment by TOoSmOotH** [comment]
Is this a VM? There seems to be CPU contention going on.
